CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 --------------------
 
+[3.7.2] 2020-09-01
```

Comment:
Might also need to change __init__.py to keep things up to date on pypi even though you don't actually need to update in platform